src/mumble/TalkingUI.cpp

```diff
@@ -569,6 +571,41 @@ void TalkingUI::mousePressEvent(QMouseEvent *event) {
 	updateUI();
 }
 
+void TalkingUI::wheelEvent(QWheelEvent *event) {
+	if (!event->modifiers().testFlag(Qt::ControlModifier)) {
+		return QWidget::wheelEvent(event);
+	}
+
+	if (!Global::get().s.talkingUI_CtrlScrollLocalVolAdj) {
+		return;
+	}
+
+	ClientUser *user = Global::get().mw->pmModel->getSelectedUser();
+	if (!user) {
+		return;
+	}
+
+	int volAdjustment = VolumeAdjustment::toIntegerDBAdjustment(user->getLocalVolumeAdjustments());
+
+	if (event->angleDelta().y() > 0) {
+		if (volAdjustment >= 30) {
+			// 30 dB is the upper limit defined for the volume slider (VolumeSliderWidgetAction.cpp).
+			return;
+		}
+
+		volAdjustment += 1;
+	} else {
+		if (volAdjustment <= -30) {
+			// -30 dB is the lower limit defined for the volume slider (VolumeSliderWidgetAction.cpp).
+			return;
+		}
+
+		volAdjustment -= 1;
+	}
+
+	user->setLocalVolumeAdjustment(VolumeAdjustment::toFactor(volAdjustment));
```

Comment:
This does set the new local volume temporarily, but it is not retained. You can test this by adjusting the volume using your method and then changing client state of the user you adjusted (e.g. deafen and immediately undeafen) 

To retain the changes we need to add:

```
Global::get().db->setUserLocalVolume(user->qsHash, user->getLocalVolumeAdjustments());
```